src/libs/ck-libs/ampi/ampiCC

```diff
@@ -89,6 +89,11 @@ do
      GLOBALS_SHARED='1'
      GLOBALS_FUNCPTR='1'
      ;;
+  -pieglobals)
```

Comment:
Are there compiler/linker options we could warn users about being incompatible with -pieglobals?

Reply:
Nothing stands out to me.